### DISCUSSION TITLE
Welcome to culfw.esp8266 Discussions!

### DISCUSSION THREAD

**Discussion by Man-fred** [Man-fred/culfw.esp8266#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Menne01** [comment]
Hallo Manfred.
du kannst dich mit Sicherheit an mich noch erinnern !?
Wir haben das Projekt vor einigen Jahren mit dem MQTT getestet. Auch über dem Fhem Adapter im iobroker lief es wegen eines Bugs nicht rund. Danach lag es brach.
Nun betreibe ich den Stick über eine umgeleitete Schnittstellle im CUN Modus und ist seit Anfang des Jahres im iobroker einwandfrei  eingebunden. Ich hatte dir ja das Gehäuse damals zur Verfügung gestellt zum aufhängen. Nun betreibe ich erfolgreich das Projekt als Stick, auf einem Fuß mit USB Anschluß. Dort hat einmal ein aktiver Wlan-Adapter incl. Antenne gesteckt.

![1672922277431-efeb5724-f761-4b37-803b-b99f1be1872c-grafik](https://github.com/Man-fred/culfw.esp8266/assets/89869516/eaba3d62-26a0-48e4-a54d-966b437f8a9c)

![1672953547637-7bbca432-fe57-4ccc-b486-93f49a84efff-grafik](https://github.com/Man-fred/culfw.esp8266/assets/89869516/e441c6ab-8204-4be1-8acc-76ff02cf4490)

Das ganze steht nun an einer günstigen Stelle im Haus., wo man alle (Fensterkontakte, Thermostaten) gut erreichen kann.
Auch das Pairing läuft damit Super.

Grüße
Dirk
